packages/vuetify/src/locale/fa.ts

```diff
@@ -31,15 +31,15 @@ export default {
   },
   datePicker: {
     ok: 'OK',
-    cancel: 'Cancel',
+    cancel: 'لغو',
     range: {
-      title: 'Select dates',
-      header: 'Enter dates',
+      title: 'تاریخ‌ها را انتخاب کنید',
+      header: 'تاریخ‌ها را وارد کنید',
     },
-    title: 'Select date',
-    header: 'Enter date',
+    title: 'تاریخ‌ را انتخاب کنید',
+    header: 'تاریخ را وارد کنید',
```

Comment:
I suggest the "Enter date" translation to be shorter, as the font is too big by default
<img width="377" alt="image" src="https://github.com/vuetifyjs/vuetify/assets/5362902/4e6bf0b7-9262-4041-aa1f-17ba2327e369">